CMSgov/mint-app
# [EASI-3217] Allow multiple roles on frontend

# EASI-3217

## Changes and Description

- Update `AddCollaborator` to take an array of new roles 
    - using multiselect over single select
- Updated `collaborator.spec.js` e2e test
- Update various tests to take in `teamRoles: ['MODEL_LEADS']`



## How to test this change

1. Add new collaborator to a model plan
2. Ensure you can select multiple roles
3. After adding, check to see a comma separated list instead of a single role





## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
